src/atlassian-connect.ts

```diff
@@ -71,16 +71,45 @@ export const connectAppDescriptor = {
 	/**
 	 * Extensions for the different parts of Jira
 	 * like links, panels, pages, permissions, workflows etc.
+	 * https://developer.atlassian.com/cloud/jira/platform/about-connect-modules-for-jira/
 	 */
 	modules: {
+		postInstallPage: {
+			key: 'figma-get-started',
+			name: {
+				value: 'Get Started',
+			},
+			url: '/static/admin',
+			conditions: [{ condition: 'user_is_admin' }],
+		},
+		webSections: [
+			{
+				key: 'figma-addon-admin-section',
+				name: {
+					value: 'Figma for Jira',
```

Comment:
/nit Would be good to use a `const` or something for both this value and `connectAppDescriptor.name` so they always match.

Reply:
Good point. Done!